<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skakavi krompir launcher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 24px; font-weight: bold; margin-bottom: 10px;</string>
     </property>
     <property name="text">
      <string>Skakavi krompir launcher</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <item>
      <widget class="QListWidget" name="instanceList">
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 14px; color: #666; margin-bottom: 10px;</string>
         </property>
         <property name="text">
          <string>Ready</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="instanceIcon">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="instanceName">
         <property name="text">
          <string>No selected instance</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>150</width>
           <height>11</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="launchBtn">
         <property name="text">
          <string>Launch</string>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="killBtn">
         <property name="text">
          <string>Kill</string>
         </property>
         <property name="icon">
          <iconset theme="process-stop"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logsBtn">
         <property name="text">
          <string>Logs</string>
         </property>
         <property name="icon">
          <iconset theme="view-list-text"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modsBtn">
         <property name="text">
          <string>Mods</string>
         </property>
         <property name="icon">
          <iconset theme="applications-system"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downloadBtn">
         <property name="text">
          <string>Download</string>
         </property>
         <property name="icon">
          <iconset theme="download"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeBtn">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset theme="edit-delete"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset theme="list-add"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
